packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -18,13 +18,22 @@ function textMarkers({
   text,
   billboard,
   background = true,
+  randomScale = false,
 }: {
   text: string,
   billboard?: ?boolean,
   background?: ?boolean,
+  randomScale?: ?boolean,
 }) {
   const radius = 10;
   const count = 10;
+  const scale = (i: number) => {
+    if (!randomScale) {
+      return { x: 1, y: 1, z: 1 };
+    }
+    // It's not really random. Otherwise, screenshot tests won't work correctly
```

Comment:
You could also use the `rng` helper function which uses a fixed seed in screenshot tests 😄 https://github.com/cruise-automation/webviz/blob/94302209e46a6787216ad9aea2ed1d8f9cc69d1c/packages/regl-worldview/src/stories/util.js#L24

Reply:
That's great! Thanks!